src/structures/GuildChannel.js

```diff
@@ -281,13 +282,12 @@ class GuildChannel extends Channel {
    * @readonly
    */
   get members() {
-    const members = new Collection();
-    for (const member of this.guild.members.cache.values()) {
-      if (this.permissionsFor(member).has('VIEW_CHANNEL', false)) {
+    return this.guild.members.cache.reduce((members, member) => {
+      if (this.permissionsFor(member)?.has(Permissions.FLAGS.VIEW_CHANNEL)) {
```

Comment:
Since we're passing a GuildMember instance, permissionsFor will always return a non-null value.
```suggestion
      if (this.permissionsFor(member).has(Permissions.FLAGS.VIEW_CHANNEL)) {
```